sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.c

```diff
@@ -387,11 +385,11 @@ void _handle_adv_l2(gnrc_netif_t *netif, _nib_onl_entry_t *nce,
                 nce->info &= ~GNRC_IPV6_NIB_NC_INFO_IS_ROUTER;
             }
         }
-#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_QUEUE_PKT) && MODULE_GNRC_IPV6
+#if MODULE_GNRC_IPV6
```

Comment:
I know this was there before, but how can this not be set if we compile a file in `gnrc/network_layer/ipv6`?

I say just remove it to declutter the file.